client/my-sites/theme/main.jsx

```diff
@@ -216,10 +219,17 @@ const ThemeSheet = React.createClass( {
 
 		const section = this.validateSection( this.props.section );
 		const priceElement = <span className="themes__sheet-action-bar-cost" dangerouslySetInnerHTML={ { __html: this.props.price } } />;
+		const siteID = this.props.selectedSite && this.props.selectedSite.ID;
 
 		return (
 			<Main className="themes__sheet">
 				{ this.renderBar() }
+				<ActivatingTheme siteId={ siteID }>
+					<ThanksModal
+						site={ this.props.selectedSite }
+						source={ 'details' }
+						clearActivated={ bindActionCreators( clearActivated, this.props.dispatch ) }/>
```

Comment:
Consider using the second argument to the `connect` call, [`mapDispatchToProps`](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options), instead:

```js
export default connect(
	( state ) => {
		/* ... */
	},
	{ clearActivated }
)( ThemeSheet );
```

(and just use `clearActivated={ this.props.clearActivated }` then -- could do the same with the other actions you `dispatch`, BTW.)